app/views/trainees/confirm_deferrals/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= render PageTitle::View.new(title: "trainees.deferral_details.confirm") %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLink.new(
+    text: 'Back',
+    href: trainee_deferral_path(@trainee)
+  ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <span class="govuk-caption-l">
+      <%= trainee_name(@trainee) %>
+    </span>
+    <h1 class="govuk-heading-l">Check deferral details</h1>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <%= render Trainees::DeferralDetails::View.new(@trainee) %>
+
+    <%= form_with url: trainee_confirm_deferral_path(@trainee), method: :put, local: true do |f| %>
+      <%= hidden_field_tag :trainee_id, @trainee.id %>
+      <%= f.govuk_submit "Defer this trainee" %>
+    <%- end -%>
+  </div>
+</div>
+
+<p class="govuk-body"><%= govuk_link_to("Cancel and return to record", edit_trainee_path(@trainee)) %></p>
```

Comment:
Remove this for now as per Ed's comment.

Reply:
I've removed the cancel link from `app/views/trainees/deferrals/show.html.erb`, I think that's the one you meant?

Reply:
I mean the one on the confirmation page, not on the show page. Once you get to the confirmation page, the deferral has already been saved to the database. So until we fix that the 'Cancel' button on the confirmation page is misleading.

Reply:
oh cool I see, sorry, i've removed the one from the confirmation page